src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double side1;
+    private final double side2;
+    private final double height;
+
+    public IsoscelesTrapezoid(String color, double side1, double side2, double height) {
+        super(color);
+        this.side1 = side1;
+        this.side2 = side2;
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
```

Comment:
The formula for the area calculation is correct, but it would be more readable if you add parentheses around the sum of the sides before dividing by 2. This will make it clear that the division by 2 applies to the sum of the sides and not just to `side2`.